x/programs/test/programs/simple_token/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "simple_token"
+version = "0.1.0"
+edition = "2021"
+
+[lib]
+crate-type = ["cdylib"]
+
+[dependencies]
+wasmlanche-sdk = { workspace = true }
+borsh = { workspace = true }
+
+[build-dependencies]
+wasmlanche-sdk = { workspace = true, features = ["build"] }
```

Comment:
Any reason not to extend the current token example? If you think it makes more sense to keep it as is, we can rename this to `token` and call the other one `simple-token` (as it is simpler).